MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -153,9 +153,8 @@ private IPlaybackManager CurrentPlaybackManager
         /// <returns></returns>
         public async Task Play(IEnumerable<IMediaFile> mediaFiles)
         {
-            var enumerable = mediaFiles as IList<IMediaFile> ?? mediaFiles.ToList();
             MediaQueue.Clear();
-            MediaQueue.AddRange(enumerable);
+            MediaQueue.AddRange(mediaFiles.ToList());
```

Comment:
Is there a need to convert this to a list? `MediaQueue.AddRange()` accepts `IEnumerable<IMediaFile>` the same way as this method does, right?